deploy/charts/harbor-container-webhook/templates/deployment.yaml

```diff
@@ -105,3 +105,7 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.topologySpreadConstraints }}
```

Comment:
I think you need `with` instead of `if` here.  Line 110 won't work right otherwise.